smriprep/workflows/outputs.py

```diff
@@ -335,10 +341,9 @@ def init_anat_derivatives_wf(
         )
 
         ds_std_t1w = pe.Node(
-            DerivativesDataSink(base_directory=output_dir, desc='preproc', keep_dtype=True,
```

Comment:
I think `keep_dtype` was a holdover from an older niworkflows, which meant we were dumping this into [the JSON](https://circle-production-customer-artifacts.s3.amazonaws.com/picard/forks/5d730ad8a6f141375eae36c5/184477333/5fb58669e54d67587de127eb-0-build/artifacts/fasttrack-derivatives/smriprep/sub-100185/anat/sub-100185_space-MNI152Lin_desc-preproc_T1w.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20201118T221137Z&X-Amz-SignedHeaders=host&X-Amz-Expires=60&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20201118%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=17876cdc3551e1af29e257d33d4c690e0cc1259b41b04f71af74dadae8ba928b).

Reply:
Yes, this is correct.